tests/extproc/envoy.yaml

```diff
@@ -59,8 +59,6 @@ static_resources:
                     typed_config:
                       "@type": type.googleapis.com/envoy.extensions.filters.http.ext_proc.v3.ExternalProcessor
                       allow_mode_override: true
-                      mutation_rules:
-                        allow_all_routing: true
```

Comment:
this was unneeded and makes the configuration diff between EG and raw envoy here smaller